src/features/profile/Menu.tsx

```diff
@@ -3,16 +3,18 @@ import {View} from 'react-native';
 
 import {CustomButton} from '@components/custom_button';
 import {MenuButton} from './MenuButton';
+import {RootScreens, RootStackParamList} from '@navigation/screens';
 
 import {styles} from './styles/menu';
 
 interface Props {
   onPress: () => void;
+  handleNavigate: (screen: RootScreens, params?: RootStackParamList) => void;
 }
 
-export const Menu: React.FC<Props> = ({onPress}: Props) => (
+export const Menu: React.FC<Props> = ({onPress, handleNavigate}) => (
   <View style={styles.container}>
-    <MenuButton title="Orders" />
+    <MenuButton title={RootScreens.Orders} onPress={handleNavigate} />
```

Comment:
@DiamondRock597 title it should be a test not string value from enum imagine if we want to add localisation....